src/scss/partials/_footer.scss

```diff
@@ -0,0 +1,36 @@
+.nav {
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 80px;
+  background-color: $color-tertiary;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+
+  &__btn-1 {
+    grid-column-start: 1;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+  }
+  &__btn-2 {
+    grid-column-start: 2;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+  }
+  &__btn-3 {
+    grid-column: 3 / 4;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    align-content: center;
+  }
+}
```

Comment:
nice structure!

Reply:
tnx, Sasha, I've try to apply the BEM methodology :)